src/Features/CSharp/Portable/Completion/KeywordRecommenders/StructKeywordRecommender.cs

```diff
@@ -15,7 +15,9 @@ internal class StructKeywordRecommender : AbstractSyntacticSingleKeywordRecommen
                 SyntaxKind.PublicKeyword,
                 SyntaxKind.PrivateKeyword,
                 SyntaxKind.ProtectedKeyword,
-                SyntaxKind.UnsafeKeyword
+                SyntaxKind.UnsafeKeyword,
+                SyntaxKind.RefKeyword,
+                SyntaxKind.ReadOnlyKeyword
```

Comment:
:bulb: Include a `,` at the end of the list, so if we ever have to add items to the end of the list the diff doesn't need to show a change on this line (see e.g. how it would have been better if the line with `UnsafeKeyword` did not need to change here).